assembly/buffer/index.ts

```diff
@@ -22,4 +22,9 @@ export class Buffer extends Uint8Array {
     result.dataLength = size;
     return result;
   }
+
+  readInt8(offset: i32) : i8 {
+    if(u32(offset) >= this.dataLength) throw new RangeError(E_INDEXOUTOFRANGE);
```

Comment:
In stdlib, which is also non-portable, this is usually written as `<u32>offset`. Might make sense to adopt the stdlib notation in the node library as well for consistency.